reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-74.md

```diff
@@ -0,0 +1,114 @@
+---
+title: What's New in PowerShell 7.4 (preview)
+description: New features and changes released in PowerShell 7.4 (preview)
+ms.date: 01/05/2023
+---
+
+# What's New in PowerShell 7.4 (preview)
+
+PowerShell 7.4-preview.1 includes the following features, updates, and breaking changes.
+
+For a complete list of changes, see the [Change Log][07] in the GitHub repository.
+
+## Breaking Changes and Improvements
+
+- Snap packages aren't available for this release
+- Nano server docker images aren't available for this release
+
+## Tab completion improvements
+
+- Fix issue when completing the first command in a script with an empty array expression
+  ([[#18355][18355]]) (Thanks @MartinGC94!)
+- Fix positional argument completion ([#17796][17796]) (Thanks @MartinGC94!)
+- Improve type inference of hashtable keys ([#17907][17907]) (Thanks @MartinGC94!)
+- Fix type inference error for empty return statements ([#18351][18351]) (Thanks @MartinGC94!)
+- Improve enumeration of inferred types in pipeline ([#17799][17799]) (Thanks @MartinGC94!)
+- Fix member completion in attribute argument ([#17902][17902]) (Thanks @MartinGC94!)
+- Improve pseudo binding for dynamic parameters ([#18030][18030]) (Thanks @MartinGC94!)
+- Add completion for values in comparisons when comparing Enums ([#17654][17654]) (Thanks
+  @MartinGC94!)
+
+## Cmdlet and engine improvements
+
+Update to WebCmdlets
+
+- WebCmdlets get **Retry-After** interval from response headers if the status code is 429
+  ([#18717][18717]) (Thanks @CarloToso!)
+- Web cmdlets set default charset encoding to UTF8 ([#18219][18219]) (Thanks @CarloToso!)
```

Comment:
Standardizing to the spacing/capitalization used above this line.
```suggestion
- WebCmdlets set default charset encoding to UTF8 ([#18219][18219]) (Thanks @CarloToso!)
```

Reply:
Will fix